src/tests/contract/app.ts

```diff
@@ -0,0 +1,27 @@
+import express from "express";
+import { accessTokenRouter } from "./routes/AccessTokenRoute";
+import { userInfoRouter } from "./routes/UserInfoRoute";
+import bodyParser from "body-parser";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Constants } from "./utils/Constants";
+
+const logger = new Logger({
+	logLevel: "INFO",
+	serviceName: "ContractTestApp",
```

Comment:
Maybe `serviceName: "F2FContractTests",` or something?

Reply:
Updated it to **F2fCriProvider** as that is the key to identify the pact file from the pactbroker. Also It is an app which will be running the mock servers for F2F cri provider endpoints.